Source/Scenes/Onboarding/1_AnalyticsPermission/AnalyticsPermissionViewModel.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  AnalyticsPermissionViewModel.swift
+//  Zupreme
+//
+//  Created by Andrei Radulescu on 09/11/2018.
+//  Copyright © 2018 Open Zesame. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+import RxCocoa
+
+// MARK: - AnalyticsPermissionNavigation
+enum AnalyticsPermissionNavigation: String, TrackedUserAction {
+    case userOptedForAnalytics
+}
+
+// MARK: - AnalyticsPermissionViewModel
+final class AnalyticsPermissionViewModel: AbstractViewModel<
+    AnalyticsPermissionNavigation,
+    AnalyticsPermissionViewModel.InputFromView,
+    AnalyticsPermissionViewModel.Output
+> {
+
+    private let useCase: OnboardingUseCase
+
+    init(useCase: OnboardingUseCase) {
+        self.useCase = useCase
+    }
+
+    override func transform(input: Input) -> Output {
+        bag <~ [
+            input.fromView.optIn.do(onNext: { [unowned self] in
+                self.useCase.optedInForAnalytics()
+                self.stepper.step(.userOptedForAnalytics)
+            }).drive(),
+
+            input.fromView.optOut.do(onNext: { [unowned self] in
+                self.useCase.optedOutForAnalytics()
+                self.stepper.step(.userOptedForAnalytics)
+            }).drive()
+        ]
+        return Output()
+    }
+}
+
+extension AnalyticsPermissionViewModel {
+    struct InputFromView {
```

Comment:
maybe `optIn` and `optOut` can be named `optInTrigger`, I have an idea of naming all `Driver<Void>` like that. I just saw that I missed this myself, e.g. in `TermsOfServiceViewModel`, `didAcceptTerms` should be named `acceptedTermsTrigger` maybe?

Reply:
Will do, I like the idea with Trigger.